packages/gatsby-plugin-sharp/src/image-data.ts

```diff
@@ -115,11 +115,40 @@ export async function generateImageData({
       : DEFAULT_BREAKPOINTS
   }
 
-  const {
-    fit = `cover`,
-    cropFocus = sharp.strategy.attention,
-    duotone,
-  } = transformOptions
+  const { fit = `cover`, duotone } = transformOptions
+
+  let cropFocus = transformOptions.cropFocus || `attention`
+
+  const cropFocusOptions = [
+    `top`,
+    `right top`,
+    `right`,
+    `right bottom`,
+    `bottom`,
+    `left bottom`,
+    `left`,
+    `left top`,
+    `north`,
+    `northeast`,
+    `east`,
+    `southeast`,
+    `south`,
+    `southwest`,
+    `west`,
+    `northwest`,
+    `center`,
+    `centre`,
+    `entropy`,
+    `attention`,
+  ]
+
+  if (typeof cropFocus !== `number` && !cropFocusOptions.includes(cropFocus)) {
+    reporter.error(
```

Comment:
What was previous behavior? Did it also fail or did it go through? I feel like we should fail during gatsby build but not during develop. What do you think?
```suggestion
    reporter.panicOnBuild(
```

Reply:
Previously, the app would crash. Also failing during build and not during develop sounds like a good idea.